components/navigation/BlogPostItem.js

```diff
@@ -22,24 +24,65 @@ export default function BlogPostItem({ post, className = '' }) {
   }
 
   return (
+    <div className={`flex flex-col border border-gray-200 rounded-lg shadow-md divide-y divide-gray-200 transition-all duration-300 ease-in-out hover:shadow-lg overflow-hidden ${className}`}>
+      <a href={post.slug} className="flex-shrink-0">
+        <img className="h-48 w-full object-cover" src={post.cover} alt="" />
+      </a>
+      <div className="flex-1 bg-white p-6 flex flex-col justify-between">
+        <div className="flex-1">
+          <Paragraph typeStyle="body-sm" textColor="text-indigo-500">
+            <span className={`inline-flex items-center px-3 py-0.5 rounded-full ${typeColors[0]} ${typeColors[1]}`}>
+              {post.type}
+            </span> 
+          </Paragraph>
+          <a href={post.slug} className="block">
+            <Heading level="h3" typeStyle="heading-sm-semibold" className="mt-2">
+              {post.title}
+            </Heading>
+            <Paragraph typeStyle="body-sm" className="mt-3">
+              <TextTruncate element="span" line={4} text={post.excerpt} />
+            </Paragraph>
+          </a>
+        </div>
+        <div className="mt-6 flex items-center">
+          <div className="relative flex-shrink-0">
+            <AuthorAvatars authors={post.authors} />
+          </div>
+          <div className="ml-3">
+            <Heading typeStyle="heading-xs-semibold" textColor="text-gray-900">
+              <span className="hover:underline">
+                {post.authors.map((author, index) => author.link ? <a key={index} alt={author.name} href={author.link}>{author.name}</a> : author.name).reduce((prev, curr) => [ prev, ' & ', curr ])}
+              </span>
+            </Heading>
+            <Paragraph typeStyle="body-sm" className="flex">
+              <time dateTime={post.date}>
+                {moment(post.date).format('MMMM D, YYYY')}
+              </time>
+              <span className="mx-1">
+                &middot;
+              </span>
+              <span>
+                {post.readingTime} min read
+              </span>
+            </Paragraph>
```

Comment:
You should remove these lines and everything should works + of course add that `break` :)

```suggestion
```

Reply:
On no!! Something weird must have happened when I merged the changes. My bad 😓

Reply:
Ok check now